ubtue_Brill.js

```diff
@@ -83,10 +83,33 @@ function postProcess(doc, item) {
 	// mark articles as "LF" (MARC=856 |z|kostenfrei), that are published as open access
 	let openAccessTag = text(doc, '.has-license span');
 	if (openAccessTag) item.notes.push('LF');
-	if (!item.itemType)
+	// numbering issues with slash due to cataloguing rule
+	if (item.issue) item.issue = item.issue.replace('-', '/');
+	let date = item.date;
+	//entry for scraping Berichtsjahr
+	let dateEntry = ZU.xpathText(doc, '//div[@class="cover cover-image configurable-index-card-cover-image"]//@title');//Z.debug(dateEntry)
+	let berichtsjahr = extractBerichtsjahr(dateEntry);//Z.debug(berichtsjahr)
+	let erscheinungsjahr = extractErscheingunssjahr(date);//Z.debug(erscheinungsjahr)
+	//
+	if (erscheinungsjahr !== berichtsjahr) {
+		item.date = extractBerichtsjahr(dateEntry);
+	} else {
+		item.date;
+	}
+  if (!item.itemType)
             item.itemType = "journalArticle";
 }
 
+function extractErscheingunssjahr(date) {
+	let publicationYear = date.trim().match(/\d{4}/)[0];
+	return publicationYear;
+}
+
+function extractBerichtsjahr(dateEntry) {
+	let extractDate = dateEntry.split('(')[1].split(')')[0];
```

Comment:
Wouldn't using a regex be more appropriate? With `split` the expression fails if there are no brackets.

Reply:
I changed the code, but i am not sure if using the match method with regex does not cause an error as well.